Events/2022-03-diverse-leung.md

```diff
@@ -0,0 +1,45 @@
+# DiveRSE Speaker Series: Mary Leung
```

Comment:
I feel like it would be better to focus on the title of the talk than the speaker?  Consider a couple of alternative ideas.  I prefer the shorter one, but I'm flexible
* DiveRSE webinar: Normalizing inclusion by embracing difference
* DiveRSE webinar: Normalizing inclusion by embracing difference by Mary Ann Leung